src/main/java/mate/academy/controller/BookController.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.controller;
+
+import java.util.List;
+import mate.academy.dto.BookDto;
+import mate.academy.dto.CreateBookRequestDto;
+import mate.academy.service.BookService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class BookController {
```

Comment:
you have the same parts of URI that may be extracted by using @RequestMapping on a class